server/gitrest/src/routes/git/refs.ts

```diff
@@ -124,7 +124,7 @@ async function patchRef(
         try {
             await externalStorageManager.write(repo, refId, patchParams.sha, true);
         } catch (error) {
-            winston.error(`External storage write failed while trying to update file 
+            winston.error(`External storage write failed while trying to update file
             ${safeStringify(error, undefined, 2)}, ${repo} / ${refId}`);
         }
```

Comment:
Note: Editor trimmed the trailing space before the newline, which doesn't really effect the output since the newline inserts a `\n` in the string.